openstack/nova/templates/api-metadata-deployment.yaml

```diff
@@ -35,44 +35,34 @@ spec:
         {{- end }}
     spec:
 {{ tuple . "nova" "api-metadata" | include "kubernetes_pod_anti_affinity" | indent 6 }}
-{{ include "utils.proxysql.pod_settings" . | indent 6 }}
+      {{- include "utils.proxysql.pod_settings" . | indent 6 }}
       terminationGracePeriodSeconds: {{ .Values.defaults.default.graceful_shutdown_timeout | add 5 }}
       hostname: nova-api-metadata
+      initContainers:
+      {{- tuple . (dict "service" (include "nova.helpers.database_services" .) "jobs" (tuple . "db-migrate" | include "job_name")) | include "utils.snippets.kubernetes_entrypoint_init_container" | indent 6 }}
```

Comment:
This is missing the `rabbitmq` dependency since the last force-push. It also includes the cell2 DB and I think we agreed in the last meeting, that it doesn't really make sense to depend on both cell1 and cell2 DB for this service, because if a node goes down, the metadata-api might not come up again, if one of the cell DBs cannot be started.

Reply:
Yes, that what before our discussion. I've changed it to just the api-db.